plugins/janus_recordplay.c

```diff
@@ -2007,6 +2008,32 @@ static void *janus_recordplay_handler(void *data) {
 			}
 			/* Tell the core to tear down the PeerConnection, hangup_media will do the rest */
 			gateway->close_pc(session->handle);
+		} else if (!strcasecmp(request_text, "pause") || !strcasecmp(request_text, "resume")) {
+			JANUS_LOG(LOG_VERB, "Record&Play: Got pause/resume request\n");
+			int pause = !strcasecmp(request_text, "pause");
+			result = json_object();
+			json_object_set_new(result, "status", json_string(pause ? "paused" : "resumed"));
```

Comment:
Shouldn't this check if the current session is in recording mode? It may be idle, or playing something instead: if playing, the session object will still have a `recording` object set, and you'd be sending broken events.